### ISSUE TITLE
implement smooth zooming

### ISSUE THREAD

**Issue by davelandry** [d3plus/d3plus-plot#57]
Would be really great to have x/y plot zooming, like in this old example: http://bl.ocks.org/stepheneb/1182434

**Comment by swingingdonut** [comment]
Hey, was this implemented?

**Comment by davelandry** [comment]
@swingingdonut nope! not yet. This issue will be closed once it has been implemented.
